dataflow/manual/content.tex

```diff
@@ -842,6 +837,35 @@ \subsection{Formal Definition of the Control-Flow Graph}
         \label{tab:nodes}
     \end{longtable}
 
+\autoref{tab:nodesWithException} shows all node types whish can possibly throw an exception and type to be thrown.
+Java class name of nodes are simplified as with \autoref{tab:nodes}.
+All of specific Exceptions type in table are in package \code{java.lang}.
+
+    \begin{longtable}{lp{0.6\linewidth}l}
+        \midrule
+        \multicolumn{2}{c}{\autoref{tab:nodesWithException}: All node types could throw Exception and types to be thrown..} \\ \\
+        \textbf{Node type} & \textbf{Exception type} \\ \midrule \endfirsthead
+
+        \textbf{Node type} & \textbf{Exception type} \\ \midrule \endhead
+        \hline \multicolumn{2}{|c|}{{Continued on next page}} \\ \hline \endfoot
+        \endlastfoot
+
+        \code{ArrayAccess} & NullPointerException, ArrayIndexOutOfBoundsException \\
+        \code{FieldAccess} & NullPointerException \\
+        \code{MethodAccess} & NullPointerException \\
+        \code{MethodInvocation} & Throwable, Types in throws clause of the signature \\
```

Comment:
I would also put the class names of the exception in `\code{}`.
The `Types` isn't a class name, so don't capitalize.